helm_deploy/values-dev.yaml

```diff
@@ -22,3 +22,4 @@ env:
   ELITE2_URI_ROOT: "https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api" 
   AUTH_URI_ROOT: "https://gateway.t3.nomis-api.hmpps.dsd.io/auth"
   JAVA_OPTS: "-Xmx512m"
+  SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://gateway.t3.nomis-api.hmpps.dsd.io/auth/.well-known/jwks.json"
```

Comment:
Have you updated `helm_deploy/keyworker-api/templates/_envs.tpl` to use these variables?

Reply:
No I forgot to put them in there in the first place :man_facepalming: Thanks Jon.